### PR TITLE
Bump some packages

### PR DESCRIPTION
**Pull request by anthonydresser**

### CODE REVIEW

File: build/azure-pipelines/darwin/continuous-build-darwin.yml [hunk 1]
```diff
@@ -60,3 +60,6 @@ steps:
     testResultsFiles: '*-results.xml'
     searchFolder: '$(Build.ArtifactStagingDirectory)/test-results'
   condition: succeededOrFailed()
+- task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0 # {{SQL CARBON EDIT}} add task
```
Comment (aaomidi) [inline build/azure-pipelines/darwin/continuous-build-darwin.yml]: Doesn't this get automatically injected anyway?

Comment (anthonydresser) [reply build/azure-pipelines/darwin/continuous-build-darwin.yml]: Kind of, it used to be auto inject into the pipeline (like you could actually see the step) now it looks like it runs silently in the background. I'm seeing how useful actualyl seeing the step is.

Comment (anthonydresser) [reply build/azure-pipelines/darwin/continuous-build-darwin.yml]: My hope is that it follows the same settings as in the component governance tab and that this step could actually be interesting to fail on for CI.

**Comment by aaomidi** [comment]
I asked the maintainer of https://github.com/microsoft/vscode-proxy-agent to update their package because of the https package issue. Should we use the updated bits too?

**Comment by anthonydresser** [comment]
Its already been updated. I bumped https://github.com/microsoft/azuredatastudio/pull/8096/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R52

File: extensions/admin-tool-ext-win/package.json [hunk 2]
```diff
@@ -74,7 +74,6 @@
     "vscode-nls": "^3.2.1"
   },
   "devDependencies": {
-    "vscode": "^1.1.10",
```
Comment (aaomidi) [inline extensions/admin-tool-ext-win/package.json]: Don't we need this package for testing?

File: extensions/dacpac/package.json [hunk 3]
```diff
@@ -60,8 +60,7 @@
     "mocha": "^5.2.0",
     "should": "^13.2.1",
     "mocha-junit-reporter": "^1.17.0",
-    "mocha-multi-reporters": "^1.1.7",
-    "vscode": "^1.1.26"
```
Comment (aaomidi) [inline extensions/dacpac/package.json]: Same here

**Comment by anthonydresser** [comment]
@kburtram @aaomidi this is ready for review.

The only functional changes are:
- Changing all instances of https-proxy-agent from 2.2.1 to 2.2.3, this is the version that vscode is using in their master
- Changing all instances of lodash and lodash.mergewith from 4.1.x to >4.1.14

These are just minor version bumps so I don't expect there to be any side effects.

**Comment by anthonydresser** [comment]
This also adds a governance check that will fail on alerts of High or Critical.

**Comment by aaomidi** [comment]
Hmm pr validation failed, lets schedule another one to see if its one off?

**Pull request merged**